liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/LooseEarApplication.java

```diff
@@ -77,7 +77,10 @@ public Element addModule(MavenProject proj, Artifact artifact, String pluginId)
         config.addDir(moduleArchive, outputDirectory, "/");
         // add manifest.mf
         File manifestFile = MavenProjectUtil.getManifestFile(proj, pluginId);
-        addManifestFileWithParent(moduleArchive, manifestFile);
+
+        String mavenProjectTargetDir = proj.getBasedir().getCanonicalPath() + "/target";
```

Comment:
Are we guaranteed it is always `target` for the output? Is that configurable?

Reply:
I'll change this to get it with `proj.getBuild().getDirectory()`